distributor-api-v1/operations.md

```diff
@@ -550,6 +550,17 @@ Gives availabilities and pricings for given date interval with product prices in
 
 ```javascript
 {
+    "RateGroups": [
+        {
+            "Id": "634d4625-e127-4282-a17d-ab51010e4deb",
+            "SettlementType": "Manual",
+            "SettlementAction": "ChargeCreditCard",
+            "SettlementTrigger": "Start",
+            "SettlementOffset": "P0M0DT0H0M0S",
+            "SettlementValue": 1.00000000,
+            "SettlementMaximumNights": null
```

Comment:
I see also `SettlementExtent: ["Nights", "Products", "AdditionalExpenses"]` in the response, should that be described too?

Reply:
Nope, that one i will delete since it is not used anywhere on FE nor BE